### DISCUSSION TITLE
How does this compare to "Peek" from Big Z Labs?

### DISCUSSION THREAD

**Discussion by ebernet** [sbarex/SourceCodeSyntaxHighlight#198] [Q&A]
Hi, I want to support developers so I purchased Peek in addition to using Syntax Highlight. Question - do either of them offer features over the others? Is there still a reason for me to use Syntax Highlight? How can I ascertain which is in use when I quicklook a file?

**Comment by sbarex** [comment]
I do not know Peek, and as it is not downloadable for free, I have no way to test it and compare it with mine.

To check if you are using my app you can change the theme. So if the preview shows the theme set in the settings you are using my software.

**Comment by ebernet** [comment]
Thank you for letting me know. I will update you when I hear back from the author of Peek. Appreciate it.
